src/hdnode.js

```diff
@@ -120,7 +118,7 @@ HDNode.fromBuffer = function(buffer, network, __ignoreDeprecation) {
   var chainCode = buffer.slice(13, 45)
   var data, hd
 
-  // 33 bytes: private key data (0x00 + k)
+    // 33 bytes: private key data (0x00 + k)
```

Comment:
@maxogden yet again.  I think this is related to `var X, Y` style declarations before a comment.